### ISSUE TITLE
XXX

### ISSUE THREAD

**Issue by YOU-TO-BE** [ZhengyuZhao/koniq-PyTorch#2]

**Comment by ZhengyuZhao** [comment]
> Why do the score values I predicted with other models be negative?

Could you please provide more details about your issue (eg., Which model and which part of the code are you using)?

**Comment by YOU-TO-BE** [comment]
`class IQAModel(nn.Module):
    def __init__(self, arch='resnest50d', num_classes=1, **kwargs):
        super(IQAModel, self).__init__()
        self.base_model = resnest50d(pretrained=True)
        self.base = nn.Sequential(*list(base_model.children())[:-1])
        # self.features = nn.Sequential(*features)
        self.fc = nn.Sequential(nn.Linear(2048, 512),
                                 nn.ReLU(inplace=True),
                                 nn.BatchNorm1d(512),
                                 nn.Dropout(p=0.25),
                                 nn.Linear(512, 256),
                                 nn.ReLU(inplace=True),
                                 nn.BatchNorm1d(256),
                                 nn.Dropout(p=0.25),
                                 nn.Linear(256, 1)
)


    def forward(self, x):

        x = self.base(x)
        x = x.view(x.size(0),-1）
        x = self.fc(x)
        return x`
